### ISSUE TITLE
feat: cart functionality

### ISSUE THREAD

**Issue by soobinrho** [soobinrho/GoodLifeFarms#3]
Now, users can select only one item at a time, and also users cannot specify quantity.

Make a cart. Also, here's an idea: keep the current buttons for each item, but whenever the user clicks on any of the button, either (a) make a popup to select the quantity (and explain to the user "Write quantity (e.g. 3 onions or 2 bulbs of garlic)"), or (b) make some beautiful / easy-to-use UI/UX such that one of existing components, such as the description of the selected item, gets transformed into a quantity-selection component.

- Relevant quote: "A user interface is well-designed when the program behaves exactly how the user thought it would. As Hillel said, everything else is commentary. All the other rules of good UI design are just corollaries." Source: https://www.joelonsoftware.com/2000/04/10/controlling-your-environment-makes-you-happy/

**Comment by adykrni** [comment]
hey, is this issue open? I would like to contribute to it, it aligns well with my experience and i can definitely help you in the long run :)

**Comment by soobinrho** [comment]
> hey, is this issue open? I would like to contribute to it, it aligns well with my experience and i can definitely help you in the long run :)

Absolutely! Opened just a few hours ago hahaha

**Comment by adykrni** [comment]
Thanks, I'll get started on it EOB. :)

**Comment by adykrni** [comment]
hey @soobinrho - For the UI concept, can I play around with the colours a bit?

**Comment by soobinrho** [comment]
> hey @soobinrho - For the UI concept, can I play around with the colours a bit?

Sure, go ahead!

**Comment by adykrni** [comment]
hey @soobinrho - here is the the link to the Figma file. 

https://www.figma.com/file/wScTJeJjjPlXIk2eTTdkzw/Good-life-Farms?type=design&node-id=0%3A1&mode=design&t=1l3czOpRR7kmb55w-1

**Comment by soobinrho** [comment]
Wow great job!

**Comment by adykrni** [comment]
thanks :) - Will you take this up for development? Also, will I get contributed for this pull request?

**Comment by soobinrho** [comment]
> thanks :) - Will you take this up for development? Also, will I get contributed for this pull request?

Absolutely! The design looks awesome! I'll code this up. Can't promise how long it will take, but when it's complete, you'll be co-authored in the commit. Thank you and great job again!

**Comment by adykrni** [comment]
Awesome :) 
Looking forward to helping you out more, if you need any design specs please feel free to reach out.
